config/run.tf

```diff
@@ -0,0 +1,107 @@
+/* [BEGIN] Cloud Run Setup */
+
+# Create a Pub/Sub topic to trigger the GCS-to-BQ service.
+resource "google_pubsub_topic" "notify_data_ingested" {
```

Comment:
I think we also need the upload_to_gcs topic here

Reply:
Good catch, thanks! I must have accidentally deleted it in the refactor.